### DISCUSSION TITLE
new css file or not

### DISCUSSION THREAD

**Discussion by nebbiadigiorno** [ExperienceLovelace/ha-floorplan#71] [Q&A] [answered]
As suggested i'll open a discussion.
I don't understand a (maybe) simple fact, but hey i am a noob :(
So, i see the example in the website and there is a real floorplan.svg, a light and a ring doorbell.
I can't understand haw can i implement te light (or the ring) in my floorplan. I have to create another "light.svg/light.css" and put it in my floorplan directory? or can i put the light.svg in my home.svg and add thhe light.css in my home.css?
Sorry for the (maybe) stupid question.......

**Comment by exetico** [comment, marked as answer]
It sounds like you're trying to add more elements to the floorplan?

We've not created a fresh guide, yet ([link](https://experiencelovelace.github.io/ha-floorplan/docs/create-svg-file/)), but here's an old guide which does the job just fine:
[Creating your own custom floorplan-file from scratch](https://github.com/pkozul/ha-floorplan/blob/master/own-floorplan-svg-file-tutorial.md)

You'll need to customize your SVG-file in Inscape or so, to add each element to the drawing.

I hope that's the actual problem, you're trying to solve :-)

**Reply by exetico** [reply]
CSS-files are not needed for each element, if you're just trying to add more elements to your floorplan.

**Reply by nebbiadigiorno** [reply]
i am trying to add a camera in my floorplan.
i add the icon in myy svg file and in the yaml. i try to click on the camera and display the video feed.

**Reply by jressel01** [reply]
show your yaml and svg file

**Reply by nebbiadigiorno** [reply]
i resolve it with your code! thanks a lot!
